### PR TITLE
ASM-7833 fix SLES mtu issue [ASM 8.3.1]

### PR DESCRIPTION
**Pull request by ksw2599**
ASM-7833 fix SLES mtu issue; default to 9000

### CODE REVIEW

File: manifests/bond/vlan.pp [hunk 1]
```diff
@@ -42,10 +42,10 @@
   $interface = $name
 
   #include '::network'
-
   $already_configured = $name in split($::interfaces, ',')
 
   if !$already_configured {
+
```
Comment (ripienaar) [inline manifests/bond/vlan.pp]: new lines after { doesnt seem to be a convention used here?

File: templates/ifcfg-suse-bond.erb [hunk 2]
```diff
@@ -3,8 +3,11 @@
 ###
 BOOTPROTO=none
 BONDING_MASTER=yes
+<% if @mtu %>MTU=<%= @mtu %>
```
Comment (ripienaar) [inline templates/ifcfg-suse-bond.erb]: either it should be a multi line if or a single line one, this is a weird combo of both :)

**Comment by ksw2599** [comment]
@ripienaar: I just made the changes.

**Comment by gavin-scott** [comment]
@ripienaar can you re-review? Looks like @ksw2599 addressed your comments.

**Comment by ripienaar** [comment]
@gavin-scott looks good, merge when ready

**Pull request merged**